addon/components/nrg-sidebar/template.hbs

```diff
@@ -1,3 +1,3 @@
 {{#ui-sidebar class="sidebar-modal menu" elementId=sidebarId onHide=(action "onHide") context=context transition="overlay" mobileTransition="overlay" silent=true}}
-  {{nrg-sidebar-menu action="onHide"}}
+  {{nrg-sidebar-menu action="onHide" clickedSidebarItem=(action "clickedLink")}}
```

Comment:
You removed the `sendAction` on this component so `action="onHide"` doesn't get used here anymore.

In the `clickedLink` action handler, you need to call the `onHide` action.

```js
this.send("onHide");
```

Reply:
This allows the sidebar to close probably when a link is clicked.